### PR TITLE
Backend/rework getBookingsByGuideId

### PR DESCRIPTION
**Pull request by MinitrueSecretary**
## Change made

- [ ] Bug fixes
- [x] New features
- [ ] Breaking changes

## Describe what you have done

- add getBookingsByGuideId in addition to getBookingsByUserId

### New Features

- 

### Fix

-

### Others

-

### CODE REVIEW

File: apps/backend/src/bookings/bookings.service.ts [hunk 1]
```diff
@@ -47,6 +49,24 @@ export class BookingsService implements IBookingsService {
     return results;
   }
 
+  async getBookingsByGuideId(
+    guideId: number,
+  ): Promise<GetBookingsByGuideIdResponse> {
+    const bookings = await this.bookingsRepo.paginateBookings({
+      offset: 0,
+      limit: 100000,
```
Comment (PongDev) [inline apps/backend/src/bookings/bookings.service.ts:57]: please use value from frontend input, too much limit will make database burden

File: apps/backend/src/bookings/bookings.service.ts [hunk 2]
```diff
@@ -63,7 +83,7 @@ export class BookingsService implements IBookingsService {
       guideId: req.guideId,
       startDate: new Date(req.startDate),
       endDate: new Date(req.endDate),
-      bookingStatus: 'waitingForGuideConfirmation',
+      bookingStatus: 'pending',
```
Comment (PongDev) [inline apps/backend/src/bookings/bookings.service.ts:86]: can you use enum like BookingStatus.pending?

**Review by PongDev — CHANGES_REQUESTED** [review]

File: apps/backend/src/bookings/bookings.service.ts (hunk 1 above)
Comment (MinitrueSecretary) [reply apps/backend/src/bookings/bookings.service.ts]: OK offset and limit added as queryparams

**Review by PongDev — DISMISSED** [review]
ok

**Review by PongDev — DISMISSED** [review]
ok

**Review by PongDev — APPROVED** [review]
Ok

**Pull request merged**